apps/src/code-studio/utils.js

```diff
@@ -48,3 +48,17 @@ export function updateQueryParam(param, value) {
   }
   window.history.pushState(null, document.title, newLocation);
 }
+
+/**
+ * We indicate that a page is Rtl by setting the dir attribute on the root element
```

Comment:
I see. In that case, I'd probably have the `setRtl()` action creator or associated reducer also update the DOM so they are guaranteed to stay in sync, and have `setRtl()` act as the only way to put it on the DOM in the first place.

So redux is effectively the source of truth, and we sync to the DOM instead of the other way around to get the css to work. In the theoretical future world where we render the entire page with react, including the rtl attribute, then having that rtl attribute populated by whatever is in the redux store would make the most sense.